atriaapp/atriacalendar/models.py

```diff
@@ -105,8 +105,18 @@ def __str__(self):
         return self.label
 
 
+# Base class for organizations that use the Atria platform
+class AtriaOrganization(models.Model):
+    org_name = models.CharField(max_length=40)
+    date_joined = models.DateTimeField(default=timezone.now)
```

Comment:
Use `auto_now_add=True` instead of `default=timezone.now`, unless you want to be able to create Orgs with something other than the current datetime?

Reply:
I'd like to leave the option of specifying a different joined date (since joining and adding the record in the database may be two separate events)